tools/licenses/lib/patterns.dart

```diff
@@ -868,7 +869,7 @@ final List<RegExp> csTemplateLicenses = <RegExp>[
     kIndent +
 
     // Some files in ANGLE prefix the license with a description of the license.
-    r'(?:BSD 2-Clause License \(http://www.opensource.org/licenses/bsd-license.php\))?' +
+    r'(?:BSD 2-Clause License \(https?://www.opensource.org/licenses/bsd-license.php\))?' +
```

Comment:
Needed for: https://github.com/search?q=repo%3AKhronosGroup%2FVulkan-ValidationLayers+%22BSD+2-Clause+License+%28https%22&type=code